tests/php/test-class-amp-meta-sanitizer.php

```diff
@@ -41,6 +41,12 @@ public function get_data_for_sanitize() {
 				'<!DOCTYPE html><html><head><meta charset="utf-8"><meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1"></head><body></body></html>',
 			],
 
+			// Trim whitespace between the properties and their values.
```

Comment:
This file should be changed so that these single-line comments for each test case are converted into array keys. This way, PHPUnit would display the exact test case that is failing.